### ISSUE TITLE
Skip plot_coverage function [Feature request]

### ISSUE THREAD

**Issue by PubuduSaneth** [dariober/cnv_facets#19]
plot_coverage function seems resource intensive and fails when cnv_facets is implemented in a nextflow process. Therefore it'll be quite useful to add a parameter to disable the plot_coverage function. 

```
[2020-02-20 15:13:02] Loading file Px.FACETS.csv.gz...
[2020-02-20 15:14:42] Plotting histogram of coverage...
command.sh: line 2:    29 Killed                  cnv_facets.R -t Px.md.recal.bam -n Px.md.recal.bam -vcf dbsnp_146.hg38.vcf.gz -o Px.FACETS
```

**Comment by dariober** [comment]
Hi- Can you comment out the lines executing `plot_coverage`? It should be on line 817-818 in cnv_facets.R. Then execute again and see what happens and please let me know. 

My guess is that if you run out of resources to make the plot than you'll run out also for the actual task of detecting CNVs, which should be more demanding. (I may be wrong though...)

Thanks
Dario

**Comment by PubuduSaneth** [comment]
Hi,

Thank you very much for the quick reply. Yes, we have commented out plot_coverage lines in the cnv_facets.R script. This is working fine... 

```
  #write(sprintf('[%s] Plotting histogram of coverage...', Sys.time()), stderr())
    # plot_coverage(rcmat= rcmat[['pileup']], rcmat_flt= rcmat_flt,
    # fname= paste0(xargs$out, '.cov.pdf'), title= paste0('Depth of coverage\n', xargs$out))
```

We are using the biocontainers cnv_facets image - quay.io/biocontainers/cnv_facets:0.15.0--r36h4b26f60_1 . So after modifying the R script in the container (from biocontainers/cnv_facets image), we commit those changes into a new docker image. This new docker image with modified cnv_facets.R script is functional in our NetFlow pipeline.  

(Potentially, in a future version) If it is possible to disable the plot_coverage function using a command-line parameter, users can directly implement that feature using biocontainers/cnv_facets docker image. 

Again, thank you so much for creating and maintaining this very useful tool.

**Comment by dariober** [comment]
Hi- I included the option `--no-cov-plot` to skip the plotting of coverage plots in v0.16.0 which is now on [bioconda](https://bioconda.github.io/recipes/cnv_facets/README.html). Thanks for reporting the issue!

**Comment by PubuduSaneth** [comment]
Dear Dario,

It looks like libcrypto.so.1.0.0 is not installed in biocontainers cnv_facets:v0.16.0. I received the following error.

```
bcftools: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
Error in checkForRemoteErrors(val) : one node produced an error:
Error in computing snp pileup. Exit code 1 from execution of ./P23993.FACETS.csv.gz_200319-031838_1d23a9455d/snp_pileup.chr1.sh
Calls: exec_snp_pileup_parallel ... clusterApply -> staticClusterApply -> checkForRemoteErrors
```
In addition to that is it possible to install fonts-conda-ecosystem fonts in the docker image. Fonts in png file (CNV plot) do not appear properly.

Again, thank you so much for developing and maintaining cnv_facets tool

**Comment by dariober** [comment]
Hi- Thanks for your patience - I love conda but sometimes... grrrrr! This issue with libcrypto seems to be resurfacing. 

I can reproduce the issue and I escalated the problem to the bioconda list (https://github.com/bioconda/bioconda-recipes/issues/21229).

**Comment by dariober** [comment]
The issue with docker and bcftools should be fixed with the new recipe 0.16.0-1. This should work now:

```
docker run -it quay.io/biocontainers/cnv_facets:0.16.0--py38r36h4b26f60_1 bash
```
